modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -1363,6 +1363,26 @@
 				Stops the function execution and returns the current suspended state to the calling function.
 				From the caller, call [method GDScriptFunctionState.resume] on the state to resume execution. This invalidates the state. Within the resumed function, [code]yield()[/code] returns whatever was passed to the [code]resume()[/code] function call.
 				If passed an object and a signal, the execution is resumed when the object emits the given signal. In this case, [code]yield()[/code] returns the argument passed to [code]emit_signal()[/code] if the signal takes only one argument, or an array containing all the arguments passed to [code]emit_signal()[/code] if the signal takes multiple arguments.
+				You can also use [code]yield[/code] to wait for a function to finish:
+				[codeblock]
+				func _ready -> void:
+				   yield(do_something(), "completed")
+				   yield(do_something_else(), "completed")
+				   print("All functions are done!")
+
+				func do_something() -> void:
+				   print("Something is done!")
+				
+				func do_something_else() -> void:
+				   print("Something else is done!")
+				
+				# prints:
+				# Something is done!
+				# Something else is done!
+				# All functions are done!
+				[/codeblock]
+				[code]completed[/code] signal is automatically created by the engine if the first argument is a function.
```

Comment:
The sentence should likely start with "The". This sentence is also not clear to me as a user: what does it mean that the engine creates a signal? Do you mean that it *emits* a signal, which I can thus connect to?